crates/sui-graphql-rpc/src/config.rs

```diff
@@ -132,8 +132,39 @@ pub struct BackgroundTasksConfig {
     pub watermark_update_ms: u64,
 }
 
-#[derive(Debug)]
-pub struct Version(pub &'static str);
+#[derive(Copy, Clone, Debug)]
+pub struct Version {
+    /// The major version is set as the year
+    pub major: &'static str,
+    /// The minor version is set as the quarter when this release happened
+    pub minor: &'static str,
+    /// The patch is just an integer that is incremented with each release
+    /// in that quarter
```

Comment:
The fact that we make breaking releases every quarter is not a feature of this versioning scheme -- we could choose to release every month, every half, every year if we wanted to -- so I would remove mentions of quarters here. The patch version is incremented for every compatible release on top of the `year.month` release.